### ISSUE TITLE
'make install' ignores the specified '--prefix'

### ISSUE THREAD

**Issue by michaelkarlcoleman** [RTcmix/RTcmix#26]
Instead of installing in the tree rooted at the specified `--prefix`, `make install` seems to put its install in the source tree.

This is with source 'git clone'ed just now, and no other configure flags.

**Comment by johgibso** [comment]
Yes, we have never supported the full capabilities of the GNU build system. Sorry about that.

**Comment by michaelkarlcoleman** [comment]
Hi @johgibso, I understand.  Is this something you'd take a patch for, if I can get around to it?
